packages/devtools_app/lib/src/analytics/analytics.dart

```diff
@@ -1,640 +1,5 @@
-// Copyright 2019 The Chromium Authors. All rights reserved.
+// Copyright 2021 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-@JS()
-library gtags;
-
-// ignore_for_file: non_constant_identifier_names
-
-import 'dart:async';
-import 'dart:html';
-
-import 'package:flutter/foundation.dart';
-import 'package:js/js.dart';
-
-import '../../devtools.dart' as devtools show version;
-import '../app.dart';
-import '../config_specific/logger/logger.dart';
-import '../config_specific/server/server.dart' as server;
-import '../config_specific/url/url.dart';
-import '../globals.dart';
-import '../performance/performance_screen.dart';
-import '../profiler/profiler_screen.dart';
-import '../ui/gtags.dart';
-import '../version.dart';
-import 'analytics_common.dart';
-import 'constants.dart' as analytics_constants;
-
-// Dimensions1 AppType values:
-const String appTypeFlutter = 'flutter';
```

Comment:
so none of these strings needed to be in the public api?

Reply:
this file was just moved, not deleted - the git diff is a little off. see _analytics_web.dart